packages/rendering/lib/aws-metrics.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+
+type Metric = {
+    send: () => void,
+};
+
+process.env.AWS_PROFILE = 'frontend';
+
+const AWS = require('aws-sdk');
+
+AWS.config.update({ region: 'eu-west-1' });
+
+// how frequently we send metrics to aws in ms
+const METRICS_TIME_RESOLUTION = 60 * 1000;
+
+const sendMetric = (m: Array) => {
+    if (m.length === 0) {
+        return;
+    }
+
+    const cloudWatchClient = new AWS.CloudWatch();
+
+    const params = {
+        MetricData: m,
+        Namespace: 'Application',
+    };
+
+    cloudWatchClient.putMetricData(params, err => {
+        if (err) {
+            console.error(err, err.stack);
+        }
+    });
+};
+
+// handles sending matrics to AWS
+
+const Collector = function collectAndSendAWSMetrics(...metrics: Metric) {
```

Comment:
For functions that are not constructor functions, prefer using arrow functions. Also prefer camel case names:

const collector = (...metrics: Metric) => {
 // ... 
}

Reply:
Gotcha. I was following https://github.com/airbnb/javascript#functions but I might have missed something elsewhere.

Reply:
Interesting! Okay, let's stick with AirBnB's recos, ignore the arrow function point  😄

Reply:
Is this better?